src/client/components/Bruincast/index.js

```diff
@@ -103,13 +103,16 @@ export const Bruincast = ({ course, warning, retrieveWarning }) => {
         {castsByCourses.map((currCourse, i) => (
           <Tabs.Panel
             id={currCourse.course.label}
+            key={currCourse.course.label}
             renderTitle={currCourse.course.label}
             selected={courseIndex === i}
           >
             <BruincastTable
+              key={currCourse.course.label}
               casts={currCourse.casts}
               selectMedia={selectMedia}
               course={currCourse.course}
+              shortname={currCourse.shortname}
```

Comment:
If I remember correctly, there should be a field under course called course.label that is basically the shortname? (I might be wrong here..)

Reply:
Ah I see, it should use currCourse.course.label instead